File: src/Suframa.Sciex.UserInterface/www/app/view-model/ManterSetorEmpresarialVM.ts
import { atividadeEconomicaVM } from './AtividadeEconomicaVM';
import { setorAtividadeVM } from './SetorAtividadeVM';

export class manterSetorEmpresarialVM {
	idSetor: number;
	codigo: number;
	descricao: string;
	tipo: number;
	observacao: string;
	status: boolean;
	codigoDivisaoAtividade: number;
	codigoGrupoAtividade: number;
	codigoClasseAtividade: number;
	atividadesEconomicasGrid: atividadeEconomicaVM[];
	setorAtividade: any[];
}

File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/ManterSetorEmpresarialVM.cs
using System.Collections.Generic;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
	public class ManterSetorEmpresarialVM
	{
		public IEnumerable<AtividadeEconomicaVM> AtividadesEconomicasGrid { get; set; }
		public int Codigo { get; set; }
		public string Descricao { get; set; }
		public int? IdSetor { get; set; }
		public string Observacao { get; set; }
		public IEnumerable<SetorAtividadeVM> SetorAtividade { get; set; }
		public bool Status { get; set; }
		public int Tipo { get; set; }
	}
}